index.rst

```diff
@@ -1,10 +1,6 @@
-:tocdepth: 1
-
-.. sectnum::
-
-.. Metadata such as the title, authors, and description are set in metadata.yaml
-
-.. TODO: Delete the note below before merging new content to the main branch.
+###########################################################################
+Preliminary study of TMA performance based on soak testing from 26-Jan-2023
+###########################################################################
```

Comment:
1. The following words are EFD commands/keywords and hence should be in italics ---> inPosition, start_slew
2. line 20 : lsst.sal.MTMount.command_trackTarget.azimuth --> *lsst.sal.MTMount.command_trackTarget.azimuth*
3. line 20: lsst.sal.MTMount.command_trackTarget.elevation --> *lsst.sal.MTMount.command_trackTarget.elevation*
4. line 20: lsst.sal.MTMount.logevent_azimuthInPosition.inPosition --> *lsst.sal.MTMount.logevent_azimuthInPosition.inPosition*
5. line 20: "..in this technote is available in" --- > ".... in this technote is available here". Put a hyper-link to the github notebook link in the word "here"
6.  line 48: Start slew ---> *Start_slew*

Reply:
you may want to check whether its start_slew or Start_slew. Craig can you help you confirm this!

Reply:
Thank you very much, all suggested changes have been made, can you check again? I tried the links and it looks like everything is working well.